cinema/models.py

```diff
@@ -51,8 +51,16 @@ def __str__(self):
 
 class MovieSession(models.Model):
     show_time = models.DateTimeField()
-    movie = models.ForeignKey(Movie, on_delete=models.CASCADE)
-    cinema_hall = models.ForeignKey(CinemaHall, on_delete=models.CASCADE)
+    movie = models.ForeignKey(
+        Movie,
+        on_delete=models.CASCADE,
+        related_name="movies"
+    )
+    cinema_hall = models.ForeignKey(
+        CinemaHall,
+        on_delete=models.CASCADE,
+        related_name="movies"
+    )
```

Comment:
Select an appropriate `related_name`, in most cases the plural form of the model that has a field is okay (e.g. `movie_sessions`).